ms/MSOper/MSConcat.cc

```diff
@@ -2387,6 +2395,65 @@ Bool MSConcat::copyPhaseCal(const MeasurementSet& otherMS,
   return True;
 }
 
+Bool MSConcat::copyEOP(const MeasurementSet& otherMS){
+  // uses newSPWIndex_p; to be called after copySpwAndPol
+
+  LogIO os(LogOrigin("MSConcat", "copyEOP"));
+
+  Bool itsEOPNull = (!itsMS.rwKeywordSet().isDefined("EARTH_ORIENTATION") || (itsMS.rwKeywordSet().asTable("EARTH_ORIENTATION").nrow() == 0));
+  Bool otherEOPNull = (!otherMS.keywordSet().isDefined("EARTH_ORIENTATION") || (otherMS.keywordSet().asTable("EARTH_ORIENTATION").nrow() == 0));
+
+  if(itsEOPNull && otherEOPNull){ // neither of the two MSs do have valid EOP tables
+    os << LogIO::NORMAL << "No valid EOP tables present. Result won't have one either." << LogIO::POST;
```

Comment:
Should probably be done with LogIO::DEBUG since this is an optional table and we don't need to record that it is not there.

Reply:
That makes sense; used LogIO::DEBUG1 since LogIO::DEBUG doesn't exist and LogIO::DEBUG1 is what is used elsewhere in this file.